### ISSUE TITLE
Increment build number so tk is built with newest zlib 1.2.12

### ISSUE THREAD

**Issue by AndresGuzman-Ballen** [AnacondaRecipes/tk-feedstock#6]
BDBA scans are detecting [CVE-2018-25032](https://nvd.nist.gov/vuln/detail/CVE-2018-25032) in the tk module. This is happening because zlib 1.2.11 had this CVE and tk links with zlib. By incrementing the build number and having tk build with zlib 1.2.12, this should go away :)

**Comment by chenghlee** [comment]
Would updating your environment to zlib 1.2.12 resolve this issue? As far as I know, `tk` should dynamically link to `libz.so`, so it's not clear to me why a rebuild is necessary.

**Comment by AndresGuzman-Ballen** [comment]
@chenghlee you're right about libz.so :) however, this issue appears specifically on WIndows with tcl86t.DLL. I should've mentioned that...my apologies.

**Comment by chenghlee** [comment]
@AndresGuzman-Ballen Gotcha; that makes sense --- thanks for clarifying!
